qdax/core/containers/repertoire.py

```diff
@@ -238,6 +238,7 @@ def add(
         batch_of_genotypes: Genotype,
         batch_of_descriptors: Descriptor,
         batch_of_fitnesses: Fitness,
+        batch_of_extra_scores: Optional[ExtraScores] = None,
```

Comment:
My understanding is that you introduce here the argument batch_of_extra_scores but not use it in the class but probably add in an other PR a child class that inherit from this one and use the batch of extra scores argument. Is that correct?
If yes, I would suggest that as we introduce the argument here we could directly use it. In that case, we could add a 'extra_scores' attribute at the MapElitesRepertoire class and update the add function so that if the argument is None nothing happens (thus it won't change current and default behavior) but that in the case where the argument exists, it is added to the grid. I think it would make sense to have this behavior by default in the MapElitesRepertoire class and will be probably useful in several examples.